resolwe_bio/processes/annotation/cuffmerge.yml

```diff
@@ -62,7 +62,7 @@
       {% for e in expressions %}
         {% for annot in gtf %}
           {% if expressions.0.source != e.source or e.source != annot.source %}
-            re-error "The input data must use Gene IDs from the same source."
+            re-error "The input data must use the same gene ID source (NCBI, UCSC, ENSEMBLE,..)."
```

Comment:
I believe it's `same gene ID database`.

Reply:
Also is it possible to make the `(NCBI, UCSC, ENSEMBLE,..)` part dynamic so that it lists only the sources actually used?

Reply:
When user needs to upload GTF or GFF file also needs to specify Gene ID source, which is then automatically referenced as source to all expression files created with this GTF or GFF.
So to keep things consistent I changed Error message in a way that the user will be able to remember where he needed to specify this source.

Reply:
No, I meant you know which sources were specified (`e.source`, `annot.source`) so instead of hardcoding `(NCBI, UCSC, ENSEMBLE,..)` you can actually list which sources were used.

Reply:
I like the idea, but `re-warning` or `re-error` takes only one argument, so it is not working by adding more than just one string.

Reply:
Barbara, this message is for end users who will not have access nor skills to change the GTF or GFF file. 

The hope is that with a precise error message, a biologist end user would at least realize they couldn't fix this in the Annotations form and contact us for help